test/e2e/framework.go

```diff
@@ -79,15 +80,6 @@ type TestData struct {
 
 var testData *TestData
 
-// Temporarily disable traffic check
```

Comment:
please don't remove these temporary removed code, we might add it back since we are now running e2e test in WCP env.